hack/make-rules/test-cmd-util.sh

```diff
@@ -2309,11 +2309,15 @@ run_configmap_tests() {
   kube::test::get_object_assert 'configmaps --namespace=test-configmaps' "{{range.items}}{{$id_field}}:{{end}}" ''
   # Command
   kubectl create configmap test-configmap --from-literal=key1=value1 --namespace=test-configmaps
+  kubectl create configmap test-binary-configmap --from-file <( head -c 256 /dev/urandom ) --namespace=test-configmaps
```

Comment:
would probably be worth setting up a e2e test that puts binary data in a configmap, and makes sure it gets plumbed to a container intact

Reply:
Added a e2e test @liggitt